index.html

```diff
@@ -6901,6 +6909,10 @@ <h2>Changes since Candidate Release of 12 December 2019</h2>
       <a href="#expansion-algorithm">Expansion algorithm</a>
       that duplicate the following step.
       This is in response to <a href="https://github.com/w3c/json-ld-api/issues/243">Issue 243</a>.</li>
+    <li>Updated step <a href="#ctd-contains-colon">16.5</a> of the
```

Comment:
For all these CR change notes, I'm guessing the hard coded section numbers are ok because this whole section will be dropped anyway?

Reply:
Well, I'm counting on minimal changes to step numbering; the section numbering should be taken care of automatically. It would be great if there were some automated way to either insert the step number, or check it. But, if we do make further changes, we'll need to look at this section for future changes.